integration-tests/arc-keep-app-classes/src/test/java/io/quarkus/arc/app/classes/tests/AppClassesNotRemovedTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.quarkus.arc.app.classes.tests;
+
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+
+@QuarkusTest
+public class AppClassesNotRemovedTest {
```

Comment:
I don't think we need an integration test for this. I'd prefer a QuarkusUnitTest in the [deployment module](https://github.com/quarkusio/quarkus/tree/master/extensions/arc/deployment/src/test/java/io/quarkus/arc/test).

Reply:
OK. I'll need to see how we pass configuration to those types of tests since that's something I've never done before.

Reply:
See for example https://github.com/quarkusio/quarkus/blob/master/extensions/arc/deployment/src/test/java/io/quarkus/arc/test/config/ConfigPropertyInjectionWithoutInjectAnnotationTest.java. 

Otherwise, it lools really good ;-)

Reply:
Fixed :)